### ISSUE TITLE
QuickStartApp using obsolete android build tools 

### ISSUE THREAD

**Issue by ekux44** [PhilipsHue/PhilipsHueSDK-Java-MultiPlatform-Android#24]
Consider updating QuickStartApp to the current Android build tools (building with gradle, targeting SDK 23, etc) as this sample app still relies on deprecated build tools from before the era of Android Studio.

**Comment by SteveyO** [comment]
I will get the SDK .jars available in Maven/Gradle first as this ticket has been open over a year and then will take a look at this.  Not 100% convinced on this as I don't want the QuickStart app tied down to any IDE or Build Tool as not all developers have gone down the Gradle/Android Studio route.  Plus we still have the Java Desktop Sample app to consider.  Will get back on this one.

**Comment by ekux44** [comment]
Yes, distributing the jars through maven/gradle is 100X more important, especially to existing users of the Hue SDK.

I don't know what percent of existing apps are still using the deprecated ADT build tool, but [gradle is now the only officially supported way to develop for Android](http://android-developers.blogspot.com/2015/06/an-update-on-eclipse-android-developer.html). Thus I believe that switching the QuickStartApp to gradle would make it quicker for new developers to get started with the SDK.
